services/web-app/pages/_middleware.js

```diff
@@ -5,58 +5,59 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import { getRunMode, RunMode } from '@carbon-platform/api/runtime'
-import cookie from 'cookie'
 import { NextResponse } from 'next/server'
 
+import sendLocalRequest from '@/utils/sendLocalRequest'
 import { isValidIbmEmail } from '@/utils/string'
 
-/**
- * Creates request options for retrieveUser api call including headers and agent if necessary
- *
- * @param {NextRequest} req getServerSideProps request object
- * @returns {RequestInit} request options
- */
-function getRequestOptions(req) {
-  const sessionCookie = req.cookies['connect.sid']
-  const cookies = sessionCookie ? cookie.serialize('connect.sid', sessionCookie) : ''
-  return {
-    headers: { cookie: cookies }
-  }
-}
+class NotAuthorizedError extends Error {}
 
 function exitWith404(req) {
   const url = req.nextUrl.clone()
   url.pathname = '/404'
   return NextResponse.rewrite(url)
 }
 
-export async function middleware(req) {
+function logIncomingRequest(req) {
+  sendLocalRequest(req, '/api/log-request', false, 'POST', {
+    logMessage: `[${new Date().toISOString()}] "${req.method} ${req.nextUrl.pathname}" "${
```

Comment:
The Logging service will automatically add a date to the request for you.

```suggestion
    logMessage: `"${req.method} ${req.nextUrl.pathname}" "${
```